autoextract/aio/retry.py

```diff
@@ -95,13 +105,22 @@ def __call__(self, retry_state: RetryCallState) -> float:
             return self.network_wait(retry_state=retry_state)
         elif _is_server_error(exc):
             return self.server_wait(retry_state=retry_state)
+        elif _is_retriable_query_error(exc):
+            try:
+                wait = exc.retry_seconds
+            except ValueError as exc:
+                logger.warning(exc)
+                return 300  # 5 minutes
+
+            return wait or self.retriable_wait(retry_state=retry_state)
```

Comment:
I think we should retry on max(wait, self.retriable_wait).

Reply:
Done.